src/main/java/EtfRecommendService/user/User.java

```diff
@@ -96,6 +96,13 @@ public void updateProfile(String nickName, Boolean isLikePrivate) {
         }
     }
 
+    public Boolean isSelfProfile(Long userId) {
```

Comment:
`null`이 될 수 있는 타입은 가능하면 사용하지 않는 것이 좋아요.
```suggestion
    public boolean isSelfProfile(Long userId) {
```